src/Makefile.am

```diff
@@ -235,23 +235,23 @@ souffle_sources = \
         interpreter/InterpreterRelation.h                  \
         parser.cc                                          \
         parser.hh                                          \
-        ram/RamCondition.h                                 \
-        ram/RamExpression.h                                \
-        ram/RamNode.h                                      \
-        ram/RamOperation.h                                 \
-        ram/RamProgram.h                                   \
-        ram/RamRelation.h                                  \
-        ram/RamStatement.h                                 \
-        ram/RamTranslationUnit.h                           \
-        ram/RamUtils.h                                     \
-        ram/RamVisitor.h                                   \
-        ram/analysis/RamAnalysis.h                         \
-        ram/analysis/RamComplexityAnalysis.cpp             \
-        ram/analysis/RamComplexityAnalysis.h               \
-        ram/analysis/RamIndexAnalysis.cpp                  \
-        ram/analysis/RamIndexAnalysis.h                    \
-        ram/analysis/RamLevelAnalysis.cpp                  \
-        ram/analysis/RamLevelAnalysis.h                    \
+        ram/Condition.h                                 \
```

Comment:
Keep the backslash alignment in the Makefile. (Or set out in a bold new direction using `text[space][space]\` and remove all alignment.)

Reply:
I will do this.